bindings/cs/Scripts/Create-and-pack-nuget.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+# This script creates 4 folder inside $packagingFolder
+# 1. libmongocrypt-all. This folder contains all platform-specific C libraries.
+# 2. csharp-v{$bindingsReleaseVersion}. After all, this folder will containts the fully ready source code.
+# 3. downloaded_mongocrypt. This folder contains the used in the libmongocrypt platform-specific C libraries.
+# 4. artifacts. After all, the generated .nupkg files will be placed here.
+#
+# Input data:
+# $libmongocryptAllUrl - this is the link on EG patch that contains all required platform-specific C libraries.
+
+packagingFolder="c:/build"
+bindingsReleaseVersion="1.2.1"
+fork="https://github.com/DmitryLukyanov/libmongocrypt.git"
+bracnhName="csharp-v1.2.1_release"
+libmongocryptAllUrl="https://mciuploads.s3.amazonaws.com/libmongocrypt/all/master/508e21f4abff9f5519e0357a63a4ad30d2c24692/libmongocrypt-all.tar.gz"
+libmongocryptAllFolder="libmongocrypt-all"
+downloadedMongocryptFolder=downloaded_mongocrypt
+csharpGitTagName=csharp-v$bindingsReleaseVersion
+
+# Start
+cd $packagingFolder
+#Download libs
+echo "Download libs"
+mkdir $libmongocryptAllFolder
```

Comment:
We should use absolute paths for these commands. Then you don't have to rely on `cd <<CORRECT_DIRECTORY>>`. This makes the script easier to understand as you don't have to keep track of your current working directory.